core/build.gradle.kts

```diff
@@ -61,4 +62,8 @@ dependencies {
   implementation(Libs.objectbox_kotlin)
   implementation(Libs.objectbox_rxjava)
   implementation(Libs.webkit)
+  // navigation
+  implementation(Libs.navigation_kotlin_fragment)
```

Comment:
so we have special ways of sharing dependencies among our modules.

If you have a look to `KiwixConfigurationPlugin` in `buildSrc`, we apply this plugin to all our modules and depending on what type of module it is ie app or library, we run what I have called Configurers which should largely resemble what you typically see in a build.gradle file.

There also is generally the option of providing an `api` instead of an `implementation` dependency from a library as what this does is say "users of this library will have access to this dependency as though it were declared in its build file"

Reply:
So adding the navigation lib to AllProjectConfigurer?

Reply:
Yeah but the plugin might be a pain to apply inside AllProject so keep the api option open

Reply:
ok cool thanks